tools/localization_analysis/scripts/plot_conversions.py

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/python3
+#
+# Copyright (c) 2017, United States Government, as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+#
+# All rights reserved.
+#
+# The Astrobee platform is licensed under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import numpy as np
+from position import Position
+from timestamped_pose import TimestampedPose
+from timestamped_velocity import TimestampedVelocity
+from value_plotter import ValuePlotter
+from vector3d_plotter import Vector3dPlotter
+import scipy.spatial.transform
+import sys
+
+# Return list of 3 lists, one each for x, y, z values in poses
+def xyz_vectors_from_poses(poses):
+    # TODO: Do this more efficiently
```

Comment:
haven't measured it, but I would suspect the way you've done it isn't bad

Reply:
done